github-actions/action.yml

```diff
@@ -0,0 +1,49 @@
+# Copyright 2020 SUSE
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: GitHub Actions Doc Generator
+description: Generates Markdown docs for GitHub Actions.
+inputs:
+  exclude:
+    description: A list of paths (one per line) to exclude from the generation.
```

Comment:
Does this support globbing? It _looks_ like it does, in which case we should document it.